filesystem/src/main/java/com/nytimes/android/external/fs/SourceFileAllReader.java

```diff
@@ -0,0 +1,13 @@
+package com.nytimes.android.external.fs;
+
+import com.nytimes.android.external.fs.filesystem.FileSystem;
+import com.nytimes.android.external.store.base.DiskAllErase;
+import com.nytimes.android.external.store.base.DiskAllRead;
+
+
+public class SourceFileAllReader extends FSAllReader implements DiskAllRead {
```

Comment:
Yes, you are right. I built it like SourceFileReader which has getRecordState method. But such a method does not make sense in AllOperations. Deleting these file.